examples/esp32spi_tcp_client.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2021 Adafruit Industries
+# SPDX-License-Identifier: MIT
+
+import board
+from digitalio import DigitalInOut
+from adafruit_esp32spi import adafruit_esp32spi
+import adafruit_esp32spi.adafruit_esp32spi_socket as socket
+import adafruit_requests as requests
+from secrets import secrets
+
+
+TIMEOUT = 5
+# adjust host and port to match server
+HOST = "192.168.10.179"
+PORT = 5000
+
+# PyPortal or similar; adjust pins as needed
+spi = board.SPI()
```

Comment:
Same here, or pull from secrets

Reply:
I agree secrets would be a good approach in general, but it isn't standard for host and port, so would require explanatory text on how to do that... so probably same approach as above